babel.config.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  presets: [
+    "@babel/preset-env",
+    "@babel/preset-react"
+  ]
+};
```

Comment:
- [ ] According to the guiding steps, this file should contain as follow
```

module.exports = {
  presets: [
    [
      '@babel/preset-env',
      {
        targets: {
          node: 'current',
        },
      },
    ],
    '@babel/preset-react',
  ],
};
```

This is from [link given to you as steps](https://betterprogramming.pub/react-with-rails-2022-bd28e1fcd355#ad0a)